docs/Gemfile.lock

```diff
@@ -253,6 +253,7 @@ GEM
     zeitwerk (2.6.0)
 
 PLATFORMS
+  ruby
```

Comment:
Hmm, I'm not sure why this change is here. I'm gonna revert this one and test again.

Reply:
Every time I run the `bundle exec` script this gets added back in so I'll leave it here.